### ISSUE TITLE
Buttons cannot be selected by letter in JAWS' Form Field List if they have the aria-pressed attribute

### ISSUE THREAD

**Issue by TestPartners** [FreedomScientific/VFO-standards-support#140]
### Summary

If a <button> element has the "aria-pressed" attribute, the button name is preceded by one or more spaces in JAWS' Form Field List (Insert+F5). The button can be selected in the list of form fields by using the Up and Down arrow keys, but it cannot be selected by typing the first letter of its name.

### Expected result

When the Form Field list is open, it should be possible to jump to any item in the list by typing the first letter of its name.

### Actual result

Nothing happens when the first letter is typed.

### Example

In this example, both buttons appear in the jaws form fields list. The Night button can be selected by typing the letter N but the Day button cannot be selected by typing the letter D.

<body>
<form>
<button aria-pressed=”true”>Day</button>
<button>Night</button>
</form>
</body>

The screenshots show how there is a space before the word "Day" but not before "Night".

![jaws form fields list 1](https://user-images.githubusercontent.com/10850299/48919598-a7ed1f00-ee8b-11e8-9149-523a36a5270d.jpg)

![jaws form fields list 2](https://user-images.githubusercontent.com/10850299/48919600-ab80a600-ee8b-11e8-9754-3f1349db5973.jpg)

### Additional Information

As far as we can tell, this does not happen with other ARIA attributes, although we have only tested a handful.

This does not seem like a major issue, but it was the subject of a complaint by our client's client.

Despite knowing the cause, we have not yet found a workaround.

#### JAWS version and build number

We tested this using JAWS 2018.1804.26 but the issue was also observed by our client and their client, who may well have been using different versions.
 
#### Operating System and version

We tested this using Windows 10

#### Browser and version:

Internet Explorer 11, Firefox 63 and Chrome 70. The behaviour is the same in all three browsers.

**Comment by lamador** [comment]
Reproduced issue using JAWS 2020.2004.66 in Chrome 81.0.4044.138, Firefox 76.0.1, and IE11 11.778.18362.0.

**Comment by stevefaulkner** [comment]
tested in JAWS  2023.2307.37 issue still present. there is a space before the button names (that have `aria-pressed`) in the form field list

**Comment by JeremiahRogersVispero** [comment]
Filed bug 148894

**Comment by JeremiahRogersVispero** [comment]
Confirmed fixed in JAWS 2026.2512.50
